src/App.js

```diff
@@ -1,14 +1,11 @@
-import React, { Component } from 'react';
 import { Calculator } from './components/Calculator';
 
-export class App extends Component { // eslint-disable-line
-  render() {
-    return (
-      <div>
-        <Calculator />
-      </div>
-    );
-  }
-}
+export const App = () => { // eslint-disable-line
+  return (
+    <div>
+      <Calculator />
+    </div>
+  );
+};
```

Comment:
Great work changing from class-based to hooks but there is a simple thing you can do with that eslint comment instead you just can fix it like this. Just remove the curly braces to get clean your code.

 `export const App = () => (
    <div>
     <Calculator />
    </div>
 );
`